### ISSUE TITLE
Conflicting isort and flake8-broken-line

### ISSUE THREAD

**Issue by Tapeline** [wemake-services/wemake-python-styleguide#3189]
### What's wrong

> Fresh installation is used. No pip packages, only `pip install wemake-python-styleguide`

When having a long import, isort suggests following format:
```py
from my_awesome_application.implementation.entities import \
    AbstractEntityFactoryImplementation
```
When running Flake8, it generates 
```
main.py:1:1: N400: Found backslash that is used for line breaking
```
If we (somehow) try to comply to N400:
```py
from my_awesome_application.implementation.entities import (
    AbstractEntityFactoryImplementation
)
```
and then run Flake8 again, it stops generating N400 warnings, but now isort complains:
```
main.py:1:1: I001 isort found an import in the wrong position
main.py:2:1: I005 isort found an unexpected missing import
main.py:3:1: I001 isort found an import in the wrong position
```

### How it should be

Either isort should be configured to agree with flake8-broken-line on line breaking or flake8-broken-line should be configured to ignore import statements

### Flake8 version and plugins

```json
{
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.12.2",
    "system": "Windows"
  },
  "plugins": [
    {
      "plugin": "darglint",
      "version": "1.8.1"
    },
    {
      "plugin": "flake8-bandit",
      "version": "4.1.1"
    },
    {
      "plugin": "flake8-broken-line",
      "version": "1.0.0"
    },
    {
      "plugin": "flake8-bugbear",
      "version": "24.12.12"
    },
    {
      "plugin": "flake8-commas",
      "version": "2.1.0"
    },
    {
      "plugin": "flake8-comprehensions",
      "version": "3.16.0"
    },
    {
      "plugin": "flake8-debugger",
      "version": "4.1.2"
    },
    {
      "plugin": "flake8-docstrings",
      "version": "1.7.0"
    },
    {
      "plugin": "flake8-eradicate",
      "version": "1.5.0"
    },
    {
      "plugin": "flake8-isort",
      "version": "6.1.1"
    },
    {
      "plugin": "flake8-quotes",
      "version": "3.4.0"
    },
    {
      "plugin": "flake8-rst-docstrings",
      "version": "0.3.0"
    },
    {
      "plugin": "flake8-string-format",
      "version": "0.3.0"
    },
    {
      "plugin": "mccabe",
      "version": "0.7.0"
    },
    {
      "plugin": "pep8-naming",
      "version": "0.13.3"
    },
    {
      "plugin": "pycodestyle",
      "version": "2.12.1"
    },
    {
      "plugin": "pyflakes",
      "version": "3.2.0"
    },
    {
      "plugin": "wemake-python-styleguide",
      "version": "0.19.2"
    }
  ],
  "version": "7.1.1"
}
```

### pip information

```
pip 23.2.1

astor==0.8.1
attrs==24.3.0
bandit==1.8.0
colorama==0.4.6
darglint==1.8.1
docutils==0.21.2
eradicate==2.3.0
flake8==7.1.1
flake8-bandit==4.1.1
flake8-broken-line==1.0.0
flake8-bugbear==24.12.12
flake8-commas==2.1.0
flake8-comprehensions==3.16.0
flake8-debugger==4.1.2
flake8-docstrings==1.7.0
flake8-eradicate==1.5.0
flake8-isort==6.1.1
flake8-quotes==3.4.0
flake8-rst-docstrings==0.3.0
flake8-string-format==0.3.0
isort==5.13.2
markdown-it-py==3.0.0
mccabe==0.7.0
mdurl==0.1.2
pbr==6.1.0
pep8-naming==0.13.3
pycodestyle==2.12.1
pydocstyle==6.3.0
pyflakes==3.2.0
Pygments==2.18.0
PyYAML==6.0.2
restructuredtext_lint==1.4.0
rich==13.9.4
setuptools==75.6.0
snowballstemmer==2.2.0
stevedore==5.4.0
typing_extensions==4.12.2
wemake-python-styleguide==0.19.2
```

### OS information

Windows 10 Professional 22H2 

_If this is not yet planned for fixing in 1.0 and you are cool with it, I can investigate possible solution and implement it_

**Comment by sobolevn** [comment]
Hi, `1.0.0` will have both `flake8-isort` and `flake8-broken-line` removed. We are using `ruff` now.

So, I am closing this issue as solved :)
